model/host/host.go

```diff
@@ -860,6 +861,44 @@ func (h *Host) GetParent() (*Host, error) {
 	return host, nil
 }
 
+// CountParentsToDecommission computes the number of hosts with containers to deco
+func CountParentsToDecommission(distroId string) (int, error) {
```

Comment:
I think this function belongs in the job rather than here because what it's doing is something pretty specific (whereas the model package contains generally useful things for interacting with the models)